README.rst

```diff
@@ -20,7 +20,7 @@ Testing
 
 After installation, you can launch the test suite::
 
-  python tests.py
+  python setup.py tests
```

Comment:
I just saw that this should be `python setup.py test`, so there is a mis-spelling here as well